src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/VxlanFlowV2Spec.groovy

```diff
@@ -518,9 +514,11 @@ class VxlanFlowV2Spec extends HealthCheckSpecification {
     def "Unable to create a vxlan flow when dst switch does not support it"() {
         given: "VXLAN supported and not supported switches"
         def switchPair = topologyHelper.getAllNeighboringSwitchPairs().find {
-            isVxlanEnabled(it.src.dpId) && !isVxlanEnabled(it.dst.dpId)
+            !isVxlanEnabled(it.src.dpId) && !isVxlanEnabled(it.dst.dpId)
```

Comment:
according to description the src swtich should support `vxlan`  
(` !isVxlanEnabled(it.src.dpId)  ->  isVxlanEnabled(it.src.dpId) `)

Reply:
fixed